x-pack/plugins/security_solution/public/cases/containers/use_update_case.tsx

```diff
@@ -89,7 +70,7 @@ export const useUpdateCase = ({ caseId }: { caseId: string }): UseUpdateCase =>
       updateKey,
       updateValue,
       updateCase,
-      version,
+      caseData,
```

Comment:
We need to know when updating the case if it has alerts. For that reason, we replaced the `version` with the whole `Case` object.